<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Button_Yul250">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Yul [250]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Coffee200">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee [200]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Tea100">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea [100]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_500">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>[500]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_100">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>[100]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_50">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>[50]</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>[10]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Change">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Ho's Vending Machine&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
